### PR TITLE
68 omniauth google

### PR DESCRIPTION
**Pull request by Danrod16**
Design and logic created

⚠️ Watch out for Edge cases bugs, is the user created in our DB? Or in an omniauth DB?
Screenshot 2023-05-25 at 17 20 39

Waiting for google verification...

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 Rails.application.routes.draw do
   mount RailsAdmin::Engine => '/admin', as: 'rails_admin'
+  devise_for :users, controllers: { omniauth_callbacks: 'users/omniauth_callbacks' }
```
Comment (christinasugiono) [inline config/routes.rb:3]: There's an error 
this line needs to be inside in line 7

File: app/javascript/controllers/tab_controller.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+// Connects to data-controller="tab"
+export default class extends Controller {
+  static targets = ["first", "second"]  
+  connect() {
+  }
+
+  // Connects to data-action="click->tab#toggle"
+  toggle(event) {
+    console.log(this.firstTarget)
```
Comment (christinasugiono) [inline app/javascript/controllers/tab_controller.js:11]: there's a console log here :D

Comment (Danrod16) [reply app/javascript/controllers/tab_controller.js:11]: Well spotted!

**Comment by Danrod16** [comment]
Add redirect to organizations/new and force user not create an organization before doing anything else.

**Comment by Danrod16** [comment]
Ready for review again, made major changes to fix roles root redirects :) @christinasugiono

File: app/javascript/controllers/tab_controller.js
Comment (Danrod16) [inline app/javascript/controllers/tab_controller.js:1]: Remove this file

File: app/views/devise/sessions/new.html.erb [hunk 3]
```diff
@@ -5,7 +5,7 @@
       <h1>Sign in</h1>
       <p>Don’t have a Wyshlist account? <%= link_to "Sign up", new_registration_path(resource_name) %></p>
 
-      <%= link_to "#", class: "btn-flat btn-ghost vw-30" do %>
+      <%= link_to user_google_oauth2_omniauth_authorize_path, method: :post, class: "btn-flat btn-ghost vw-30", data: { turbo: false } do %>
```
Comment (Danrod16) [inline app/views/devise/sessions/new.html.erb:8]: Change to Button_to

**Pull request merged**